arrow-core-data/src/main/kotlin/arrow/core/Option.kt

```diff
@@ -544,11 +544,14 @@ sealed class Option<out A> : OptionOf<A> {
     value
   }
 
-  fun show(SA: Show<A>): String = fold({
-    "None"
-  }, {
-    "Some(${SA.run { it.show() }})"
-  })
+  fun show(SA: Show<A>): String = fold(
+    {
+      "None"
+    },
+    {
+      "Some(${SA.run { it.show() }})"
+    }
+  )
```

Comment:
Hi @danimontoya :raised_hands: I see several format changes. What if we move these format changes to a single PR (with just format changes) to detect what changes come from the integration in an easier way?

Reply:
If they come from the new version of KTLint so we could create a PR for that upgrade before.

Reply:
yeah, makes totally sense! I'll do that, thanks for the suggestion! 😄